openassessment/workflow/api.py

```diff
@@ -342,11 +342,9 @@ def _get_workflow_model(submission_uuid):
         raise AssessmentWorkflowRequestError("submission_uuid must be a string type")
 
     try:
-        workflow = AssessmentWorkflow.objects.get(submission_uuid=submission_uuid)
-    except AssessmentWorkflow.DoesNotExist:
-        raise AssessmentWorkflowNotFoundError(
-            u"No assessment workflow matching submission_uuid {}".format(submission_uuid)
-        )
+        workflow = AssessmentWorkflow.get_by_submission_uuid(submission_uuid)
```

Comment:
I agree, we should avoid hitting `AssessmentWorkflow.objects` directly if at all possible. :+1: to this change